### PR TITLE
[LICENSE] - Rename gpl => LICENSE.

### PR DESCRIPTION
**Pull request by ivandrofly**
Will add new column **MIT** https://help.github.com/articles/adding-a-license-to-a-repository/

### CODE REVIEW

**Comment by xylographe** [comment]
```
PS> git grep -lF gpl.txt
build.bat
installer/Subtitle_Edit_installer.iss
src/SubtitleEdit.csproj
```

File: src/SubtitleEdit.csproj [hunk 1]
```diff
@@ -1355,6 +1355,9 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="Resources\Transparent Background.png" />
+    <Content Include="..\LICENSE.txt">
+      <Link>LICENSE.txt</Link>
```
Comment (niksedk) [inline src/SubtitleEdit.csproj]: Is the "Link" really needed?

Comment (ivandrofly) [inline src/SubtitleEdit.csproj]: > Is the "Link" really needed?

Yeah, I think so. It was added by VS2015 automatically.
Maybe it's a new tag in VS2015

Comment (ivandrofly) [inline src/SubtitleEdit.csproj]: Or maybe exclude LICENSE.txt from VS since it's not going to be *changed?

Comment (xylographe) [inline src/SubtitleEdit.csproj]: IIRC the `<Link>` item metadata is used when the file is located outside the project directory. But it is not really necessary here. I would suggest item `<Content Include="..\LICENSE.txt" />` without metadata (i.e., replace `..\gpl.txt` with `..\LICENSE.txt`).

Comment (ivandrofly) [inline src/SubtitleEdit.csproj]: Updated. @niksedk

**Pull request merged**